### ISSUE TITLE
a11y: Fix keyboard navigation and focus display for charts grid

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#12783]
a11y Issue: 169

The charts grid under apps and marketplace is not accessible with the keyboard.

This should be fixed for the grid control - this is used in other areas that need to be checked and fixed:

1. Charts
2. Secret: Create
3. Cluster Management - Create
4. 

![image](https://github.com/user-attachments/assets/f30fcf6f-2340-4666-97d4-6d62cc4cbb2a)

**Comment by yonasberhe23** [comment]
Needs manual validation

**Comment by IsaSih** [comment]
ONLY the tests in the charts menu and cluster management create pass in  Rancher | v2.11-3d729db8a848fb2523bbc2f3f21aa5367dcdcc69-head
Dashboard | master df18acc
Rancher CLI | v2.11.0-rc.1

Keyboard navigation is just working in the "More Info" link of the Create Secrets grid as seen in the screen recording

https://github.com/user-attachments/assets/7b9b313a-1b68-4c99-995e-77492f741e28

**Comment by IsaSih** [comment]
Reopening the issue for the Create Secrets to be fixed

**Comment by aalves08** [comment]
@IsaSih you are totally right. I forgot about the Secrets part, apparently 😅 I've created a PR https://github.com/rancher/dashboard/pull/13414 to address this.

**Comment by IsaSih** [comment]
Create Secrets page works as expected in
Rancher | v2.11-1ec5c626b1f615573b7f7269d4eb67ddb8c82749-head
Dashboard | master c15d79e
Rancher CLI | v2.11.0-rc.1
